docs/examples.html

```diff
@@ -1,127 +1,127 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

```

Comment:
It seems that way doesn't it? I feel like I might've had some kind of justification for this but I can't think of it now. Pushed a new diff that fixes this.